### DISCUSSION TITLE
Disable spellcheck in code blocks?

### DISCUSSION THREAD

**Discussion by wbconnor** [davidmyersdev/octo#207] [Ideas]
Red underlines in my code blocks are distracting. I'm wondering how easy it would be to disable spell check in code blocks.

**Comment by hnasheralneam** [comment]
I also had a problem with this. I think it would depend if he’s using a spellcheck tool or the browser spellcheck, though it is most probably the former, which could complicate things. Should I submit an issue?

**Reply by davidmyersdev** [reply]
It actually uses the system spellcheck. This can be fixed by adding a `spellcheck="false"` attribute to each of the code decorations.
https://github.com/voracious/ink-mde/blob/d1da8752e1cf37a458ddfbd0a3e410f2436776f5/src/vendor/extensions/code.ts#L14-L19

**Reply by davidmyersdev** [reply]
This has been fixed in the most recent release.
